Helper/Cart.php

```diff
@@ -283,7 +283,6 @@ public function isCheckoutAllowed()
      * Load Quote by id
      * @param $quoteId
      * @return \Magento\Quote\Model\Quote
-     * @throws NoSuchEntityException
```

Comment:
Because it is not thrown anymore since your (justified and reasonable )changes to how this method works. It returns false when not found and we should update the code when the exception is expected.